alternate/Tenkuu Danzai Skelter Heaven (alternate).ass

```diff
@@ -26,213 +28,220 @@ Style: OPTL,Monotype Corsiva,22,&H00F8F091,&H000000FF,&H00000000,&H7F000000,-1,0
+Dialogue: 0,0:12:59.76,0:13:01.06,Default,,0,0,0,,That's good.
+Dialogue: 0,0:13:01.06,0:13:03.43,Default,,0,0,0,,Taichou's order is absolute.
+Dialogue: 0,0:13:03.43,0:13:06.43,Default,,0,0,0,,You really are always honest, huh?
+Dialogue: 0,0:13:07.03,0:13:07.70,Default,,0,0,0,,Eh?
+Dialogue: 0,0:13:19.41,0:13:21.75,Default,,0,0,0,,Rin, get a hold of yourself!
+Dialogue: 0,0:13:25.42,0:13:27.12,Default,,0,0,0,,I'm okay, I'm okay.
+Dialogue: 0,0:13:27.12,0:13:28.12,Default,,0,0,0,,I can still go!
+Dialogue: 0,0:13:28.45,0:13:30.49,Default,,0,0,0,,Rin... no.
+Dialogue: 0,0:13:30.49,0:13:31.76,Default,,0,0,0,,All units, retreat.
+Dialogue: 0,0:13:33.12,0:13:35.23,Default,TLC,0,0,0,,We will finish it off.
+Dialogue: 0,0:13:35.23,0:13:38.80,Default,,0,0,0,,When its weakness is known, we'll attack this guy and destroy it!
+Dialogue: 0,0:13:38.80,0:13:40.73,Default,,0,0,0,,Funagai! Are you crazy?
+Dialogue: 0,0:13:42.23,0:13:43.90,Default,,0,0,0,,This was just work.
+Dialogue: 0,0:13:45.67,0:13:48.71,Default,,0,0,0,,Thinking that, I somehow did it.
+Dialogue: 0,0:13:49.47,0:13:50.28,Default,,0,0,0,,But...
+Dialogue: 0,0:13:51.54,0:13:52.68,Default,,0,0,0,,They...
+Dialogue: 0,0:13:53.31,0:13:54.41,Default,,0,0,0,,were laughing.
+Dialogue: 0,0:13:56.11,0:13:58.38,Default,,0,0,0,,I am... I am...
+Dialogue: 0,0:14:08.29,0:14:09.26,Default,TM,0,0,0,,Ah!
+Dialogue: 0,0:14:13.47,0:14:15.93,Default,,0,0,0,,What are you doing, Rin?
+Dialogue: 0,0:14:15.93,0:14:20.31,Default,,0,0,0,,I didn't follow your orders because I was made like that!
+Dialogue: 0,0:14:20.81,0:14:23.94,Default,,0,0,0,,It's because I loved you!
+Dialogue: 0,0:14:26.85,0:14:27.88,Default,,0,0,0,,That's it.
+Dialogue: 0,0:14:29.01,0:14:30.95,Default,,0,0,0,,I'm going on ahead!
+Dialogue: 0,0:14:36.39,0:14:37.26,Default,,0,0,0,,Don't go!
+Dialogue: 0,0:14:37.89,0:14:39.79,Default,,0,0,0,,Stop!
+Dialogue: 0,0:14:57.98,0:14:59.18,Default,,0,0,0,,I got in!
+Dialogue: 0,0:15:01.18,0:15:03.15,Default,,0,0,0,,Rin-san, I'm going!
+Dialogue: 0,0:15:06.18,0:15:10.02,Default,,0,0,0,,Skelter fighter Skelter wave has penetrated the target.
+Dialogue: 0,0:15:11.66,0:15:14.39,Default,,0,0,0,,What... what is this?
+Dialogue: 0,0:15:22.10,0:15:24.20,Default,,0,0,0,,This is the controlling core.
+Dialogue: 0,0:15:24.90,0:15:28.67,Default,,0,0,0,,The voice is entering into my head.
+Dialogue: 0,0:15:31.58,0:15:35.45,Default,,0,0,0,,It's true, there are humans that can't control emotions.
+Dialogue: 0,0:15:35.45,0:15:38.95,Default,,0,0,0,,And for the repeating breaking love, the end can't be seen.
+Dialogue: 0,0:15:40.32,0:15:44.42,Default,,0,0,0,,But that's just one side of humans.
+Dialogue: 0,0:15:45.62,0:15:47.56,Default,,0,0,0,,We need to beat you.
+Dialogue: 0,0:15:49.60,0:15:50.08,Default,TM,0,0,0,,No. 
+Dialogue: 0,0:15:50.86,0:15:52.93,Default,,0,0,0,,It's  not for humanity's sake.
+Dialogue: 0,0:15:53.26,0:15:55.70,Default,,0,0,0,,Its to protect one person who needs be protect!
```

Comment:
I don't think SkelterSubs is _that_ bad at English. I think we should also keep the ただの in there somehow. I'm unsure of whether ため needs to be represented literally in here as well, but that might sound too normal, especially since it was also used in the previous line.

Consider one of these:
```suggestion
Dialogue: 0,0:15:53.26,0:15:55.70,Default,,0,0,0,,Its to protect just one person who needs to be protected!
```
```suggestion
Dialogue: 0,0:15:53.26,0:15:55.70,Default,,0,0,0,,Its to protect just one person who needs protecting!
```